package.json

```diff
@@ -61,13 +61,13 @@
   },
   "devDependencies": {
     "after": "0.8.2",
-    "connect-redis": "3.4.2",
+    "connect-redis": "^6.0.0",
```

Comment:
This version update seems to break the example that uses it. The example needs to be updated to use the changed API (this particular example is not included in the test suite currently, sorry!).